demo-openid/src/Issuer.ts

```diff
@@ -117,15 +118,33 @@ function getCredentialRequestToCredentialMapper({
 }: {
   issuerDidKey: DidKey
 }): OpenId4VciCredentialRequestToCredentialMapper {
-  return async ({ holderBinding, credentialConfigurationId, credentialConfiguration, authorization }) => {
+  return async ({
+    holderBinding,
+    credentialConfigurationId,
+    credentialConfiguration,
+    authorization,
+    issuanceSession,
+  }) => {
+    // Example of how to use the the access token information from the chained identity server.
+    let authorizedUser = authorization.accessToken.payload.sub
+    if (
+      issuanceSession.chainedIdentity?.externalAccessTokenResponse?.id_token &&
+      typeof issuanceSession.chainedIdentity?.externalAccessTokenResponse?.id_token === 'string'
+    ) {
+      const claims = decodeJwt(issuanceSession.chainedIdentity.externalAccessTokenResponse.id_token)
+      if (typeof claims.email === 'string') {
+        authorizedUser = claims.email
+      }
```

Comment:
Can we verify this ID Token? To make sure the demo doesn't skip this important security step.

Thinking of it i think we need a callback to handle (optionally) the authorization response from the external authorization server.  In case there's just the normal `authorization_code` code flow, we can exchange it for an access token, but in the case of an id token, we either need to verify that id token, or we need to let the credo user verify it in a callback